config/index.js

```diff
@@ -0,0 +1,8 @@
+export default {
```

Comment:
I've given you access to the existing firebase

We should create zdev-dev. zdev-project-query is for a specific feature. What if we add more features!